connected care: add features docs (#20303)
pingcap/docs: tidb-cloud/connected-slack-ticket-creation.md

---
title: Create Tickets and Subscribe to Ticket Updates in Slack
summary: Introduces detailed information about the Slack ticket creation and update subscription.
---

# Create Tickets and Subscribe to Ticket Updates in Slack

> For customers subscribed to the Enterprise [support plan](/tidb-cloud/connected-care-detail.md), TiDB Cloud provides a ticket bot called **PingCAP Support Bot** in [Slack](https://slack.com/), to support quick ticket creation.

Reviewer: ```suggestion
For customers subscribed to the **Enterprise** [support plan](/tidb-cloud/connected-care-detail.md), TiDB Cloud provides a ticket bot called **PingCAP Support Bot** in [Slack](https://slack.com/), to support quick ticket creation.
```

## Create a support ticket

> In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will send you a message with a **Raise Request** button.

Reviewer 2: ```suggestion
In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will send you a message with a **Raise request** button.
```

> In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will respond with a thread containing a `Raise Request` button.

Reviewer: ```suggestion
In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will respond with a thread containing a **Raise Request** button.
```

> In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will respond with a thread containing a **Raise Request** button.

Author: ```suggestion
In the Slack support channel, you only need to mention **PingCAP Support Bot** and describe the problem in a message. Then, the ticket bot will send you a message with a **Raise Request** button.
```

![slack-ticket-creation-1](/media/tidb-cloud/connected-slack-ticket-creation-1.png)

> Click **Raise request** to open a form, fill it out according to the problem, and then click `Create` to submit the ticket.

Reviewer 2: ```suggestion
Click **Raise request** to open a form, fill it out according to the problem, and then click **Create** to submit the ticket.
```

> Click the `Raise request` button to open a form, fill it out according to the problem, and then click `Create` to submit the ticket.

Reviewer: ```suggestion
Click the **Raise request** button to open a form, fill it out according to the problem, and then click `Create` to submit the ticket.
```

![slack-ticket-creation-2](/media/tidb-cloud/connected-slack-ticket-creation-2.png)

> After successfully creating the ticket, the bot will reply with the ticket link. You can click the link to view the ticket details in the support portal.

Reviewer 2: ```suggestion
After successfully creating the ticket, the bot will reply with the ticket link. You can click the link to view the ticket details in the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals).
```

![slack-ticket-creation-3](/media/tidb-cloud/connected-slack-ticket-creation-3.png)

![slack-ticket-creation-4](/media/tidb-cloud/connected-slack-ticket-creation-4.png)

## Subscribe to ticket updates

> The Enterprisesupport plan supports subscriptions to ticket updates in Slack. When a support engineer responds to a ticket, a ticket update message will appear in the Slack channel. The message includes information such as the ticket title, ticket link, and last comment content.

Reviewer: ```suggestion
The **Enterprise** support plan supports subscriptions to ticket updates in Slack. When a support engineer responds to a ticket, a ticket update message will appear in the Slack channel. The message includes information such as the ticket title, ticket link, and last comment content.
```

![slack-ticket-creation-5](/media/tidb-cloud/connected-slack-ticket-creation-5.png)

## FAQs

> 1. How can I check the status of my ticket?

Reviewer 2: ```suggestion
- How can I check the status of my ticket?
```

    Log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/user/requests) with the email address used to create the ticket. You can view all historical tickets and their status for the current account.

## Contact support

> For help or questions, please contact our support team at <support@pingcap.com>.

Reviewer: ```suggestion
For help or questions, please contact our support team at <a href="mailto:support@pingcap.com">support@pingcap.com</a>.
```

> 1. How can I check the status of my ticket?
> 
> Log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/user/requests) with the email address used to create the ticket. You can view all historical tickets and their status for the current account.
> 
> ## Contact support
> 
> For help or questions, please contact our support team at <support@pingcap.com>. We are happy to help you.

Reviewer: ```suggestion
1. How can I check the status of my ticket?

    Log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/user/requests) with the email address used to create the ticket. You can view all historical tickets and their status for the current account.

## Contact support

For help or questions, please contact our support team at <support@pingcap.com>.
```
